metricbeat/tests/system/test_processors.py

```diff
@@ -93,16 +97,16 @@ def test_dropevent_with_condition(self):
         )
         metricbeat = self.start_beat()
         self.wait_until(
-            lambda: self.output_count(lambda x: x >= 1),
+            lambda: self.output_count(lambda x: x >= 2),
             max_timeout=15)
 
         metricbeat.kill_and_wait()
 
         output = self.read_output(
             required_fields=["@timestamp"],
-        )
-        for event in output:
-            assert float(event["system.process.cpu.total.pct"]) >= 0.001
+        )[1]
+        print(output)
```

Comment:
Is this leftover from debugging?

Reply:
Yah, still trying to fix the windows tests.